app/views/goodsProfile/GoodsRecordsView.scala.html

```diff
@@ -56,40 +60,126 @@
     @if(form.errors.nonEmpty) {
         @govukErrorSummary(ErrorSummaryViewModel(form))
     }
+    <a href="#goodsRecords" class="govuk-skip-link" data-module="govuk-skip-link">@messages("goodsRecords.skip.record")</a>
+    <a href="#main-content" class="govuk-skip-link" data-module="govuk-skip-link">@messages("goodsRecords.skip.main")</a>
+<div class="govuk-width-container">
+    <main class="govuk-main-wrapper" id="main-content">
+        <div class="govuk-grid-row"><h1 class="govuk-heading-l">@messages("goodsRecords.title")</h1></div>
 
-<div class="govuk-grid-column-two-thirds">
-    <h1 class="govuk-heading-l">@messages("goodsRecords.title")</h1>
+        <div class="govuk-grid-row">
 
-    <p class="govuk-body">@messages("goodsRecords.p1")</p>
+            <div class="govuk-grid-column-one-third custom-box">
+                <style>
+                    .custom-box {
+                      box-shadow: inset 0 0 0 1px #b1b4b6;
+                      margin-top: -1px;
+                      margin-bottom: -1px;
+                      padding-top: 20px;
+                      padding-right: 20px;
+                      padding-bottom: 15px;
+                      padding-left: 20px;
+                               }
+                </style>
 
-    <p class="govuk-body">@messages("goodsRecords.p2")</p>
+                <h2 class="govuk-heading-m">@messages("goodsRecords.search")</h2>
+                <p class="govuk-body"> @messages("goodsRecords.search.1")</p>
 
-    <ul class="govuk-list govuk-list--bullet">
-      <li>'@messages("goodsRecords.li1")'</li>
-      <li>'@messages("goodsRecords.li2")'</li>
-    </ul>
+                @formHelper(action = controllers.goodsProfile.routes.GoodsRecordsController.onSearch(page), 'role -> "search") {
 
-    <p class="govuk-body"><a href="@controllers.goodsRecord.routes.CreateRecordStartController.onPageLoad().url" class="govuk-link">@messages("goodsRecords.link1")</a></p>
-
-        @formHelper(action = controllers.goodsProfile.routes.GoodsRecordsController.onSearch(page)) {
-            <div style="display: flex; align-items: flex-end;">
-                @govukInput(
+                <div class="govuk-label govuk-label--s">
+                    @govukInput(
                     InputViewModel(
-                        field = form("value"),
-                        label = LabelViewModel(messages("goodsRecords.label")).withCssClass("govuk-!-font-weight-bold")
+                    field = form("value"),
+                    label = LabelViewModel(messages("goodsRecords.label")).withCssClass("govuk-!-font-weight-bold")
                     )
-                )
+                    )
+
+                </div>
+                }
+                <fieldset class="govuk-fieldset govuk-!-margin-top-4 govuk-!-margin-bottom-4" id="filters">
```

Comment:
We should be using the govuk checkboxes components for the below as that will handle fieldset and checkboxes

I recommend downloading this: https://github.com/hmrc/play-frontend-govuk-extension as it will show the Scala twirl code needed on the design website below

https://design-system.service.gov.uk/components/checkboxes/#checkboxes-default-0-play